scout/server/blueprints/panels/views.py

```diff
@@ -37,7 +48,34 @@ def api_panels(panel_name):
 @templated("panels/panels.html")
 def panels():
     """Show all panels for a user"""
-    if request.method == "POST":  # Edit/create a new panel and redirect to its page
+
+    # Add search box and add results if applicable
+    panels_found_compiled = []
+    search_string = ""
+    if request.method == "POST" and request.form.get("search_for"):
+        # Query db for panels containing the search string
+        # TODO: removed escape of form input, sonarcloud will complain
+        hgnc_symbols = ""
+        search_string = escape(request.form.get("search_for"))
+        try:
+            hgnc_symbols = parse_raw_gene_ids([search_string])
+        except ValueError:
+            flash(
+                "Provided gene info could not be parsed! " "Please allow autocompletion to finish.",
+                "warning",
+            )
+        # TODO: dont crash after error
+        LOG.debug("SEARCH: {}".format(search_string))
+        LOG.debug("HGNC: {}".format(hgnc_symbols))
+
+        for hgnc_id in hgnc_symbols:
```

Comment:
This is not actual any more since you are using the autocompletion no?

If you try searching by more that one symbol the form doesn't submit:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/28093618/185887082-915c7d30-c893-4dc0-b735-3588c8643a47.png">

<img width="522" alt="image" src="https://user-images.githubusercontent.com/28093618/185887163-607919d5-f3c5-4b4f-a089-8428ff774a03.png">

Reply:
Right now search is only supported for one gene at a time. Though the input check is done in javascript, the backend is not prepared to handle multiple search words separated by commas. 

-Do we/should we support this? 


![image](https://user-images.githubusercontent.com/1150065/185890678-e46c71e4-d6ae-44ce-84ae-160f94575e19.png)